nyaa/templates/layout.html

```diff
@@ -44,8 +44,174 @@
 		<![endif]-->
 	</head>
 	<body>
+		{% set nyaa_cats = [

```

Comment:
Just a comment...
I really think the categories should be loaded from the database when the server starts.
There is already a table for main_categories and sub_categories, they just need an extra column for `title`,
and a class that loads them then combines main & sub.
And last but not least, a function to load them lazily, as a list of dicts, like it's set up here.